.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
To samo, ten katalog nie powinien tutaj trafić

Reply:
jak ja mam teraz zmienić ten request żeby mergować do develop?

Reply:
Zamknąć ten request i utworzyć nowy?